biztalk/core/how-to-enable-receive-location-fault-tolerance.md

```diff
@@ -23,17 +23,22 @@ ms.custom: "biztalk-2020"
 
 ## Overview
 
-To make your environment highly available, you can create two or more host instances for each receiving host. In such environment, receive workloads will be distributed between different host instances. For example, you can use a Network Load Balancer (NLB), to distribute receiving workload for HTTP adapter. But if a receive adapter encounters error in one host instance, it may completely disable the receive location and other host instances running the receive adapter will stop processing incoming workloads. For example, you can create three host instances (A, B and C) to run HTTP receive adapter. If HTTP receive adapter fails in host instance A, say, due to disk full error, it can completely disable the receive location. Host instance B and C will stop receiving message though these hosts have no issues.
+To make your environment highly available (HA), you can create two or more host instances for each receiving host. In a HA environment, receive workloads are distributed between different host instances.
 
-**Starting with BizTalk Server 2020 and newer**, you can configure receive locations to recover from transient errors instead of getting completely disabled. On error, receive locations will be disabled in the host instance where it is faulted and BizTalk will attempt to recover it in certain configurable interval. The receive location will continue running in other host instances.
+You can use a Network Load Balancer (NLB), to distribute the receiving workload for the HTTP adapter. If a receive adapter has an error in one host instance, then it may completely disable the receive location. Other host instances running the receive adapter stop processing incoming workloads.
```

Comment:
```
You can use a Network Load Balancer (NLB), to distribute the receiving workload for the HTTP adapter
```

HTTP adapter is just an example, but this feature applies to all adapters that support HA.
From the above changes, it seems to suggest that the feature is specific to HTTP.

Can we reword to something like:

```
For example, say you create three host instances (A, B anc C) to run your HTTP receive location and use a 
Network Load Balancer (NLB) to distribute the receiving workload across these host instances. If the HTTP 
receive adapter fails in host instance A, such as a disk full error, it can completely disable the receive 
location. Host instances B and C also stop receiving messages, even though these host instances don't have 
any issues.
```

Reply:
Your reword looks great.